CoreQualityControlView/test/unit/src/au/gov/asd/tac/constellation/views/qualitycontrol/widget/DefaultQualityControlAutoButtonNGTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.qualitycontrol.widget;
+
+import au.gov.asd.tac.constellation.graph.StoreGraph;
+import au.gov.asd.tac.constellation.graph.schema.Schema;
+import au.gov.asd.tac.constellation.graph.schema.SchemaFactoryUtilities;
+import au.gov.asd.tac.constellation.graph.schema.analytic.AnalyticSchemaFactory;
+import au.gov.asd.tac.constellation.views.qualitycontrol.QualityControlEvent;
+import au.gov.asd.tac.constellation.views.qualitycontrol.QualityControlEvent.QualityCategory;
+import au.gov.asd.tac.constellation.views.qualitycontrol.daemon.QualityControlState;
+import au.gov.asd.tac.constellation.views.qualitycontrol.rules.QualityControlRule;
+import java.awt.GraphicsEnvironment;
+import java.util.ArrayList;
+import java.util.List;
+import javafx.embed.swing.JFXPanel;
+import javafx.scene.control.Tooltip;
+import static org.mockito.Mockito.*;
```

Comment:
I think it would be better to import only the bits of the library that are being used rather than whole thing